Build fails when trying to build with conda, boost library not found

Dear All. I am trying to do standalone compilation with conda on Ubuntu 24.04. I am following the instructions given here:

https://cms-analysis.github.io/HiggsAnalysis-CombinedLimit/latest/#standalone-compilation-with-conda

Unfortunately I get the following error when I execute the build command

```bash
...anaconda3/envs/combine/bin/../lib/gcc/x86_64-conda-linux-gnu/7.5.0/../../../../x86_64-conda-linux-gnu/bin/ld: cannot find -lboost_filesystem
...anaconda3/envs/combine/bin/../lib/gcc/x86_64-conda-linux-gnu/7.5.0/../../../../x86_64-conda-linux-gnu/bin/ld: cannot find -lboost_program_options
...anaconda3/envs/combine/bin/../lib/gcc/x86_64-conda-linux-gnu/7.5.0/../../../../x86_64-conda-linux-gnu/bin/ld: cannot find -lboost_system
collect2: error: ld returned 1 exit status
make: *** [Makefile:127: build/lib/libHiggsAnalysisCombinedLimit.so] Error 1
```

The error is related to the boost library not being found. This is weird, because it gets installed, as it is listed in conda_env.yml. Please, see attachments for logs.
[make.txt](https://github.com/user-attachments/files/16101531/make.txt)
[mamba.txt](https://github.com/user-attachments/files/16101533/mamba.txt)

--- a/conda_env.yml
+++ b/conda_env.yml
@@ -9,6 +9,6 @@ dependencies:
   - gsl
   - tbb
   - vdt
-  - boost
+  - boost-cpp
   - pcre
   - eigen

